eth2/state_processing/src/per_block_processing/tests.rs

```diff
@@ -10,7 +10,7 @@ use types::test_utils::{
 use types::*;
 
 pub const NUM_DEPOSITS: u64 = 1;
-pub const VALIDATOR_COUNT: usize = 10;
+pub const VALIDATOR_COUNT: usize = 64;
```

Comment:
These tests were failing on the newly introduced empty attestation bitfield check, so I had to bump this